keepassxc-browser/keepassxc-browser.js

```diff
@@ -1515,23 +1518,26 @@ cip.fillInFromActiveElementTOTPOnly = function(suppressWarnings) {
     const el = document.activeElement;
     cipFields.setUniqueId(jQuery(el));
     const fieldId = cipFields.prepareId(jQuery(el).attr('data-cip-id'));
-    const pos = _loginId;
-
-    if (pos >= 0 && cip.credentials[pos]) {
-        // Check the value from stringFields (to be removed)
-        const $sf = _fs(fieldId);
-        if (cip.credentials[pos].stringFields && cip.credentials[pos].stringFields.length > 0) {
-            const sFields = cip.credentials[pos].stringFields;
-            for (const s of sFields) {
-                const val = s["KPH: {TOTP}"];
-                if (val) {
-                    cip.setValue($sf, val);
+
+    browser.runtime.sendMessage({
+        action: 'page_get_login_id'
+    }).then((pos) => {
+        if (pos >= 0 && cip.credentials[pos]) {
+            // Check the value from stringFields (to be removed)
+            const $sf = _fs(fieldId);

```

Comment:
Why the $?  _sf() is also not very readable.

Reply:
It's a common practice to use `$` when a variable is something jQuery uses directly. I've already removed these in the WIP No jQuery PR. I'll change this to a more readable form.